translations/harbour-contrac-de.ts

```diff
@@ -306,60 +305,60 @@
     </message>
     <message id="contrac-settings_he_server_details">
         <source>Server details</source>
-        <translation type="unfinished"></translation>
+        <translation>Server-Details</translation>
     </message>
     <message id="contrac-settings_co_download-coverage">
         <source>Download coverage</source>
-        <translation type="unfinished"></translation>
+        <translation>Download-Abdeckung</translation>
     </message>
     <message id="contrac-settings_he_daily-update">
         <source>Daily update</source>
-        <translation type="unfinished"></translation>
+        <translation>Tägliche Aktualisierung</translation>
     </message>
     <message id="contrac-settings_ts_perform-daily-update">
         <source>Perform daily update</source>
-        <translation type="unfinished">Tägliche Aktualisierung durchführen</translation>
+        <translation>Tägliche Aktualisierung durchführen</translation>
     </message>
     <message id="contrac-settings_vb_approx_update-time">
         <source>Approximate update time</source>
-        <translation type="unfinished"></translation>
+        <translation>Ungefähre Updatezeit</translation>
     </message>
     <message id="contrac-settings_vb_approx_update-time-description">
         <source>The app must be running for the update to trigger</source>
-        <translation type="unfinished"></translation>
+        <translation>Die App muss ausgeführt werden, um die Aktualisierung auslösen zu können</translation>
     </message>
     <message id="contrac-guid_instructions">
         <source>Please enter the 38-digit GUID that you were given.</source>
-        <translation type="unfinished"></translation>
+        <translation>Bitte geben Sie die 38-stellige GUID ein, die Sie erhalten haben.</translation>
     </message>
     <message id="contrac-guid_invalid-character">
         <source>Invalid entry. Please check your entry.</source>
-        <translation type="unfinished">Ungültiger Eintrag, bitte überprüfen Sie diesen.</translation>
+        <translation>Ungültiger Eintrag, bitte überprüfen Sie diesen.</translation>
     </message>
     <message id="contrac-guid_invalid-guid">
         <source>Invalid GUID, please check your entry.</source>
-        <translation type="unfinished"></translation>
+        <translation>Ungültige GUID, bitte überprüfen Sie Ihren Eintrag.</translation>
     </message>
     <message id="contrac-main_bu_enter-guid">
         <source>Enter GUID</source>
-        <translation type="unfinished"></translation>
+        <translation>GUID eingeben</translation>
     </message>
     <message id="contrac-guid_he_guid_entry">
         <source>GUID entry</source>
         <oldsource>TAN entry</oldsource>
-        <translation type="unfinished">TAN-Eingabe</translation>
+        <translation>GUID-Eintrag</translation>
```

Comment:
"GUID" ist wahrscheinlich schon im Englischen ein sehr technischer Begriff, aber in der deutschen Sprache weiß der DAU damit sicher nix mehr anzufangen. Wir fändest du "Test-ID-Eintrag"?

Reply:
Ich habe eine Weile recherchiert, GUID wird von den meisten deutschen Stellen, die darauf eingehen als "Identifikationsnummer" übersetzt, was ja auch technisch dem "Unique Identifier" entspricht, wie wäre das?

Die Zertifikatsausteller/Formulardesigner nehmen allgemein an, dass man den QR-Code scannt und nicht die Nummer eintippen muss, die ID wird daher auch nicht hervorgehoben oder mit einem feststehenden Begriff versehen (á la "GUID: 1234..", sondern einfach nur die Nummer), daran kann man sich also nicht halten.

Reply:
Mir wäre wegen des Kontexts wichtig, dass dort nicht nur id, sondern auch klar steht, dass es mit dem Test zusammenhängt. Jede:r versteht was auch in D was eine ID ist, aber nur "Identifikationsnummer" ohne den Kontext-präfix find ich sehr uneindeutig. Mit "Test-Identifikationsnummer" wäre ich genauso zufrieden. Wenn ich Zeit, Lust und ich nicht Probleme mit der API bekomme, werde ich den QR-Code scan bald implementieren.

Reply:
Entry/Eintrag hatte ich wortwörtlich übersetzt, ohne den genauen Kontext in der App zu kennen (habe Contrac erst jetzt das erste Mal auf meinem XA2 ausgeführt, mit meinem Jolla 1 ging's ja vorher nicht). Wie wäre es, einfachheitshalber als Überschrift "GUID-Eingabe" zu nutzen, und bei den zwei Sätzen dann jeweils "Identifikationsnummer" statt GUID?

Reply:
Ich finde hier, dass auf der Startseite der App es sehr wichtig ist, dass der Kontext auf dem Button explizit genannt wird, ansonsten ist es für den User nicht ersichtlich, wofür diese Funktion ist. Der "Test-" Präfix ist meiner Meinung nach zwingend notwending. Bei GUID-Eingabe versteht niemand, der sich nicht eingelesen hat, was das bedeuten. Nur weil ich beim Entwickeln des Features so miese Begriffe im Englischen verwendet habe, muss der Fehler ja nicht wiederholt werden.

Reply:
Du bringst da etwas durcheinander, der "TeleTAN eingeben"-Button wird in meinem Locale-Update gar nicht verändert, es geht hier um die Überschrift, nachdem man diesen Button gedrückt hat, und die Eingabe-Seite erscheint ("TAN-Eingabe").

Die TANs, GUIDs und QR-Codes und die "TANs" im Locale stiften bei mir mittlerweile Verwirrung. Wenn ich mir die aktuelle Corona-Warn-App anschaue, gibt es einmal die Möglichkeit QR-Codes für alle Arten von Zertifikaten/Tests zu scannen (was bei Contrac ja bisher nur manuell geht, und nur bei positiven Tests), oder eine TAN bei positivem PCR-Test einzugeben/telefonisch anzufordern. Es gibt also einmal die GUID, und auch weiterhin eine TAN. So wie ich es verstanden habe, wird in Contrac die bisherige "TeleTAN" durch die GUID/Identifikationsnummer ersetzt, gibt es die TAN-Abfrage dann immer noch, als zweiten Schritt, oder wie stellt sich das nach aktuellem dar? Oder wird die "TAN" ganz rausgeschmissen?

Reply:
@bionade24

Reply:
Die `GUID` ist der Wert des QR-Codes, der auch unten rechts auf den Zetteln steht, die man bekommt, wenn man sich hat testen lassen.  Ich habe das Anfang dieses Jahres beigetragen, seitdem kann contrac aus Testergebnisse abrufen und wenn die positiv sind, entsprechend die Schlüssel hochladen. Nur QR-code scan fehlt, da als ich begonnen habe das zu Programmieren, es noch keine QR-Code API in Sailfish gab.

Reply:
Ja, die Funktion der GUID ist klar, das beantwortet aber nicht inwiefern TANs noch eine Rolle innerhalb Contracs spielen.

Reply:
Die TeleTAN ist für Leute die warum auch immer halt nicht über einen Testzentrum-Test ein positives Ergebnis bekommen haben sondern sich so eine TeleTAN bei der Hotline besorgen. Oder gibt's die Funktion in der CWA nicht mehr?

Reply:
Aha, also gibt es dann doch zwei getrennte Eingabemasken? Einmal GUID, einmal TeleTAN? Die TAN gibt es bei CWA, wie oben erwähnt, für PCR-Testsergebnisse, dann mit dem Umweg über die Hotline, die einem die TAN durchgibt.

Reply:
TAN und TeleTAN sind auch nicht das gleiche: https://github.com/corona-warn-app/cwa-documentation/blob/master/images/solution_architecture/interaction_flow_verification.svg